src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
 public class SalaryInfo {
+    private static final SimpleDateFormat FORMATTER = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String separator = "\\s";
+        Map<String, Integer> employeesSalary = new LinkedHashMap<>();
+        Date startDate;
+        Date endDate;
+
+        try {
+            startDate = FORMATTER.parse(dateFrom);
+            endDate = FORMATTER.parse(dateTo);
+        } catch (ParseException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
remove try-catch

Reply:
The tests will fail if I remove the try-catch block due to related problems in SalaryInfoTests